### ISSUE TITLE
Missing files from the project, and the localhost website was left in

### ISSUE THREAD

**Issue by Th4tITGuy** [Acumatica/acumatica-boxstorageprovider#8]
When opening the solution, we get the following error:

C:\Git\acumatica-box-windows-sdk-v2\Box.V2\Box.V2.csproj : error  : The project file could not be loaded. Could not find a part of the path 'C:\Git\acumatica-box-windows-sdk-v2\Box.V2\Box.V2.csproj'.  C:\Git\acumatica-box-windows-sdk-v2\Box.V2\Box.V2.csproj

http://localhost/Panova : error  : Error opening web http://localhost/Panova. Unable to open the Web site 'http://localhost/Panova'.  The Web site 'http://localhost/Panova' does not exist.

**Comment by gmichaud** [comment]
Just edit the .sln file manually and replace /Panova with whatever is your local dev site. You also need to have this solution somewhere on your drive: https://github.com/Acumatica/acumatica-box-windows-sdk-v2

**Comment by Th4tITGuy** [comment]
Yes I know how to fix it, I am commenting so this project can be cleaned up so people don't have to when they pull it. If the second project is required, it should also be mentioned in the read.md file, at least in the prerequisites.

Prerequisites
Acumatica 5.3 or higher
A Box.com account (personal or enterprise)
